runtime/onert/core/src/ir/OperationValidator.cc

```diff
@@ -84,6 +84,18 @@ void OperationValidator::visit(const operation::AddN &node)
   }
 }
 
+void OperationValidator::visit(const operation::ArgMax &node)
+{
+  const auto input_index(node.getInputs().at(operation::ArgMax::Input::INPUT));
```

Comment:
How about adding checking axis type too?

Reply:
I've updated to check axis type. PTAL